scripts/tests/chiptest/accessories.py

```diff
@@ -33,141 +34,162 @@
-        ok = True
-        for accessory in self.__accessories.values():
-            # make sure to do kill() on all of our apps, even if some of them returned False
-            ok = accessory.kill() and ok
-        return ok
+    @with_accessories_lock
+    def kill_all(self):
+        # Make sure to do kill() on all of our apps, even if some of them returned False
+        results = [accessory.kill() for accessory in self._accessories.values()]
+        return all(results)
 
+    @with_accessories_lock
     def start(self, name, args):
-        accessory = self.__accessories[name]
-        if accessory:
+        if accessory := self._accessories[name]:
             # The args param comes directly from the sys.argv[2:] of Start.py and should contain a list of strings in
             # key-value pair, e.g. [option1, value1, option2, value2, ...]
-            options = self.__createCommandLineOptions(args)
-            return accessory.start(options)
+            return accessory.start(self._create_command_line_options(args))
         return False
 
+    @with_accessories_lock
     def stop(self, name):
-        accessory = self.__accessories[name]
-        if accessory:
+        if accessory := self._accessories[name]:
             return accessory.stop()
         return False
 
+    @with_accessories_lock
     def reboot(self, name):
-        accessory = self.__accessories[name]
-        if accessory:
+        if accessory := self._accessories[name]:
             return accessory.stop() and accessory.start()
         return False
 
-    def factoryResetAll(self):
-        for accessory in self.__accessories.values():
+    @with_accessories_lock
+    def factory_reset_all(self):
+        for accessory in self._accessories.values():
             accessory.factoryReset()
 
-    def factoryReset(self, name):
-        accessory = self.__accessories[name]
-        if accessory:
+    @with_accessories_lock
+    def factory_reset(self, name):
+        if accessory := self._accessories[name]:
             return accessory.factoryReset()
         return False
 
-    def waitForMessage(self, name, message, timeoutInSeconds=10):
-        accessory = self.__accessories[name]
-        if accessory:
+    @with_accessories_lock
+    def wait_for_message(self, name, message, timeoutInSeconds=10):
+        if accessory := self._accessories[name]:
             # The message param comes directly from the sys.argv[2:] of WaitForMessage.py and should contain a list of strings that
             # comprise the entire message to wait for
             return accessory.waitForMessage(' '.join(message), timeoutInSeconds)
         return False
 
-    def createOtaImage(self, otaImageFilePath, rawImageFilePath, rawImageContent, vid='0xDEAD', pid='0xBEEF'):
+    def create_ota_image(self, otaImageFilePath, rawImageFilePath, rawImageContent, vid='0xDEAD', pid='0xBEEF'):
         # Write the raw image content
         with open(rawImageFilePath, 'w') as rawFile:
```

Comment:
We can just do `Path(rawImageFilePath).write_text(rawImageContent)` here

Reply:
Fixed